resources/views/livewire/portal/shop/product-detail.blade.php

```diff
@@ -0,0 +1,131 @@
+<div class="flex flex-col gap-8 h-full" x-init="$watch('$wire.productForm.product_cross_sellings', () => {showCrossSelling = $wire.productForm.product_cross_sellings[0]?.id})" x-data="{showMeta: null, showCrossSelling: null}">
+    <x-spinner />
+    <div class="relative grid sm:flex gap-8 h-full">
+        <div class="w-full sm:w-1/2 flex justify-items-center gap-4">
+            <div class="w-full">
+                <div class="w-full">
+                    <img src="{{ route('icons', ['name' => 'photo']) }}" x-bind:src="$wire.productForm.cover_url" alt="{{ $productForm->name }}" class="w-full object-contain max-h-96 rounded-lg" />
+                </div>
+                <div class="flex gap-2 justify-start">
+                    @foreach($productForm->media as $media)
```

Comment:
Fix size of the image, if images with different sizes are uploaded, the image selection "jumps". Just as a side note, I don't expect it to be fixed immediately.